beetsplug/subsonicupdate.py

```diff
@@ -40,60 +39,56 @@ def __init__(self):
         super(SubsonicUpdate, self).__init__()
 
         # Set default configuration values
-        config['subsonic'].add({
+        self.config.add({
```

Comment:
The reason this was previously `config['subsonic']` instead of `self.config` is so that the configuration could show up in a shared `subsonic:` section, hypothetically to be used by other plugins that also want Subsonic configuration. Rather the same way that the `mbcollection` plugin uses a section called `musicbrainz`:
https://github.com/beetbox/beets/blob/eb6bbaeee8c2cb9ac143cbdc41be0f4c62d1fb94/beetsplug/mbcollection.py#L61

Reply:
Ahh. Okay, interesting. I was confused because it seemed that other configs were being initialized differently. This makes sense. I also was confused because my tests were not overwriting what was defined as default in this file.